src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -66,12 +69,7 @@ class RatedDisabilityList extends React.Component {
     }
     return (
       <div className="vads-u-margin-y--5">
-        <AlertBox
-          headline={headline}
-          content={content}
-          status={status}
-          isVisible
-        />
+        <AlertBox content={content} status={status} isVisible />
```

Comment:
`headline` is optional for the `AlertBox` component and it uses an `h3`. Using an `h2` with utility classes in the above content block addresses single level header progression.